qemu/tests/single_driver_install.py

```diff
@@ -179,6 +180,16 @@ def run(test, params, env):
                    "by hwids: '%s'" % device_hwid)
 
     error_context.context("Verifying target driver", test.log.info)
+
+    ext_host = utils_net.get_ip_address_by_interface(ifname="%s" % params.get("netdst"))
+    test.log.info("ext_host of netkvm adapte is %s" % ext_host)
+    guest_ip = vm.get_address(1)
```

Comment:
I'm curious how to determine the ip owner? rtl8139  or virtio-net ?

Reply:
I use [get_address](https://github.com/avocado-framework/avocado-vt/blob/master/virttest/virt_vm.py#L784) function call to get my IP, and its parameter "1" comes from the "single_driver_install.cfg" file, where it is defined as [nic_model_nic2](https://github.com/autotest/tp-qemu/blob/master/qemu/tests/cfg/single_driver_install.cfg#L56).

Reply:
As our discussion, I changed "Index as 1" to "Name as nic2" now. This also works. Please review it when you are free. Thanks in advance.